aodntools/timeseries_products/hourly_timeseries.py

```diff
@@ -217,6 +217,35 @@ def get_nominal_depth(nc):
     return nominal_depth
 
 
+def get_contributors(files_to_agg):
```

Comment:
This is exactly the same code you already have in `aggregated_timeseries.py`, so it is not necessary to repeat it here. All you have to do is import it (at the top of the file):
```python
from aodntools.timeseries_products.aggregated_timeseries import get_contributors
```

Reply:
done. Perhaps you need to adjust the path of the aggregated_timeseries package.

Reply:
Actually, what I suggested above was correct, as an absolute import. Alternatively, you need to explicitly make it a relative import with a leading '.' (see [PEP 328](https://www.python.org/dev/peps/pep-0328)):
```python 
from .aggregated_timeseries import get_contributors
```
Currently, the unittest fails because it can't find the module: https://travis-ci.org/aodn/python-aodntools/jobs/611674191#L299